tensorflow/java/src/main/java/org/tensorflow/Output.java

```diff
@@ -22,10 +22,10 @@
  * <p>An Output is a symbolic handle to a tensor. The value of the Tensor is computed by executing
  * the {@link Operation} in a {@link Session}.
  *
- * <p>By implementing the {@link Input} interface, instances of this class could also be passed
+ * <p>By implementing the {@link Operand} interface, instances of this class could also be passed
```

Comment:
I think it's okay to refer to the `op` package, since that is where this will be used. If you don't care about the `op` package (like in Android distributions perhaps), then you don't need to care about the fact that "it can also act as something you don't care about " :)